components/FilterForm/FilterForm.js

```diff
@@ -1,10 +1,40 @@
-import React from "react";
+import React, { useState } from "react";
+import FilteredCarsList from "./FilteredCarsList";
+import data from "../../data.json";
 
 function FilterForm() {
+  const [list, setList] = useState(data);
+
+  function handleSubmit(event) {
+    event.preventDefault();
+    //fetch input data
+    const formData = new FormData(event.target);
+    const inputData = Object.fromEntries(formData);
+    //destructing
+    const { CountryOfManufacture, BodyType, Fuel, minPrice, maxPrice } =
+      inputData;
+
+    //if the number is in the range
+    function between(x, min, max) {
```

Comment:
The function can be defined outside of the `FilterForm` component, as it doesn't need any local variables to determine its output.

Pros: it doesn't get defined on every form submission!